pkg/constant/constants.go

```diff
@@ -19,4 +19,12 @@ const (
 	Referer                       = "Referer"
 	UserAgent                     = "User-Agent"
 	EnvSubdomainValidationEnabled = "SUBDOMAIN_VALIDATION_ENABLED"
+	DisabledDevStackTracerEnv     = "prod,live,demo,trial"
```

Comment:
why name it DisabledDevStackTracerEnv, is it an env?

Reply:
yes, to mark it as a constant which consist environment name. Do you have any suggestion to make it better @angelo-chan?

Reply:
Hi @angelo-chan , I just thought if we don't need this if we only depend on RealmLives checking here 
_realmLives, realmLiveExists := os.LookupEnv("REALM_LIVE")_
If we can't find this value, so we'll let `DevStackTraceable` be true

Reply:
the realmLives should have default values.

Reply:
okay, I think we don't need this constant then @angelo-chan 
I'll update the code

Reply:
I see... Okay, so we don't need `DisabledDevStackTracerEnv` constant then.
I'll update the code @angelo-chan